capstone-app/src/styles.scss

```diff
@@ -2,3 +2,6 @@
 
 html, body { height: 100%; }
 body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+
+/* Importing Bootstrap SCSS file. */
+@import '~bootstrap/scss/bootstrap';
```

Comment:
I'm a little worried this is going to mess up a lot of our styling. Can you import this locally to your private style sheet for onboarding?

Reply:
I haven't noticed any style changes

Reply:
Does it hurt anything to move it?

Reply:
I vote no as well. In fact, I don't think you should use bootstrap at all personally.

Reply:
So from what I understand, Bootstrap was being used for this carousel component:
https://ng-bootstrap.github.io/#/components/carousel/examples

Material does not have a carousel component so Sofia was using this one. However, now the component appears to be commented out. Is the carousel still needed, Sofia? If not then let's remove all the bootstrap imports and dependencies

Also I did a bit of digging and found another carousel implementation using Angular that might be helpful if we lean against using Bootstrap and still want a carousel: https://www.npmjs.com/package/angular-responsive-carousel